server/api/db/sqldb/db.py

```diff
@@ -3661,6 +3661,7 @@ def store_background_task(
         project: str,
         state: str = mlrun.common.schemas.BackgroundTaskState.running,
         timeout: int = None,
+        error: str = None,
```

Comment:
make sure you trim error to [:255] or db will fail that (db schemas is limited to 255 chars)